src/server/scripts/ExilesReach/zone_exiles_reach.cpp

```diff
@@ -5920,6 +5925,886 @@ class quest_resizing_the_situation_horde : public quest_resizing_the_situation
+                case EVENT_CAPTAIN_GARRICK_RIDE_BOAR_TALK_TO_HENRY:
+                {
+                    Player* player = me->GetOwner()->ToPlayer();
+                    if (!player)
+                        break;
+
+                    Conversation* conversation = Conversation::CreateConversation(CONVERSATION_CAPTAIN_GARRICK_RIDE_BOAR_QUEST_HENRY, player, *player, player->GetGUID(), nullptr, false);
+                    if (!conversation)
+                        break;
+
+                    conversation->AddActor(ACTOR_ALLIANCE_CAPTAIN, 0, me->GetGUID());
+                    conversation->AddActor(ACTOR_HENRY_GARRICK_PRISONER, 1, _henryGUID);
+                    conversation->Start();
+
+                    me->GetMotionMaster()->Clear();
+                    me->GetMotionMaster()->MovePoint(POINT_HENRY_POSITION, HenryPosition);
+                    _events.ScheduleEvent(EVENT_CAPTAIN_GARRICK_RIDE_BOAR_HENRY_QUESTGIVER, 18s);
+                    break;
+                }
+                case EVENT_CAPTAIN_GARRICK_RIDE_BOAR_HENRY_QUESTGIVER:
+                {
+                    if (Creature* henry = ObjectAccessor::GetCreature(*me, _henryGUID))
+                        henry->SetNpcFlag(UNIT_NPC_FLAG_QUESTGIVER);
+                    break;
+                }
+                case EVENT_CAPTAIN_GARRICK_RIDE_BOAR_EXIT_BOAR_CONVERSATION:
+                {
+                    Unit* owner = me->GetOwner();
+                    if (!owner)
+                        break;
+
+                    Player* player = owner->ToPlayer();
+                    if (!player)
+                        break;
+
+                    Conversation* conversation = Conversation::CreateConversation(CONVERSATION_CAPTAIN_GARRICK_RIDE_BOAR_QUEST_EXIT, player, *player, player->GetGUID(), nullptr, false);
+                    if (!conversation)
+                        break;
+
+                    conversation->AddActor(0, 0, player->GetGUID());
+                    conversation->AddActor(ACTOR_ALLIANCE_CAPTAIN, 1, me->GetGUID());
+                    conversation->Start();
+                    break;
+                }
+                default:
+                    break;
+            }
+        }
+    }
+private:
+    EventMap _events;
+    ObjectGuid _henryGUID;
+    bool _inCombat;
+};
+
+enum GiantBoar
+{
+    EVENT_GIANT_BOAR_SIZE_ONE         = 1,
+    EVENT_GIANT_BOAR_SIZE_TWO         = 2,
+    EVENT_GIANT_BOAR_SIZE_THREE       = 3,
+    EVENT_GIANT_BOAR_SIZE_FOUR        = 4,
+    EVENT_GIANT_BOAR_EJECT_PASSENGERS = 5,
+    EVENT_GIANT_BOAR_UNROOT           = 6,
+
+    SOUND_ENLARGE_BOAR                = 157516,
+    SOUND_SHRINK_BOAR                 = 157517
+};
+
+// Entry 156267 - Giant Boar
+struct npc_giant_boar_vehicle_q55879 : public VehicleAI
+{
+    npc_giant_boar_vehicle_q55879(Creature* creature) : VehicleAI(creature)
+    {
+        Initialize();
+    }
+
+    void Initialize()
+    {
+        me->SetOrientation(0.844224f);
```

Comment:
move this line directly into constructor (where this method is called), then drop this method